src/components/UtsLeftRight.tsx

```diff
@@ -5,7 +5,7 @@ interface UtsLeftRightProps {
 	centerVertically: boolean;
 	className?: string;
 	left: React.ReactElement;
-	right?: React.ReactElement;
+	right?: React.ReactElement | null;
```

Comment:
Does it have to be optional if it can be null?

Reply:
You're right, it's better to leave it just as optional, since this allows us to omit the property entirely if we don't need it.